User_Service/Client/cliet.go

```diff
@@ -0,0 +1,257 @@
+package ClientPackage
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	environmentVariable "github.com/E-Furqan/Food-Delivery-System/EnviormentVariable"
+	payload "github.com/E-Furqan/Food-Delivery-System/Payload"
+)
+
+type Client struct {
+	BaseUrl                   string
+	ProcessOrderURL           string
+	OrderPort                 string
+	AuthPort                  string
+	GenerateResponseUrl       string
+	RefreshTokenUrl           string
+	UserOrderUrl              string
+	ViewOrderDetailsUrl       string
+	ViewOrderWithoutDriverUrl string
+	DriverOrderUrl            string
+}
+
+func NewClient() *Client {
+	return &Client{}
+}
+
+func (client *Client) SetEnvValue(envVar environmentVariable.Environment) {
+	client.BaseUrl = envVar.BASE_URL
+	client.ProcessOrderURL = envVar.PROCESS_ORDER_URL
+	client.OrderPort = envVar.ORDER_PORT
+	client.AuthPort = envVar.AUTH_PORT
+	client.GenerateResponseUrl = envVar.GENERATE_TOKEN_URL
+	client.RefreshTokenUrl = envVar.REFRESH_TOKEN_URL
+	client.UserOrderUrl = envVar.User_ORDERS_URL
+	client.ViewOrderDetailsUrl = envVar.VIEW_ORDER_DETAIL_URL
+	client.ViewOrderWithoutDriverUrl = envVar.VIEW_ORDER_WITHOUT_DRIVER_URL
+	client.DriverOrderUrl = envVar.DRIVER_ORDERS_URL
+}
```

Comment:
we are getting a new object of client, uninitialized, and initializing separately. Why?